README.md

```diff
@@ -27,8 +27,8 @@ To get started with the Quickstart application follow these steps:
 2. Type in an identity and click on "Generate Access Token" from the [Testing Tools page](https://www.twilio.com/console/video/runtime/testing-tools).
   <img width="700px" src="images/quickstart/generate_access_token.png"/>
 
-3. Paste the Access Token into the VideoActivity.java.
-  <img width="700px" src="images/quickstart/activity_access_token.png"/>
+3. Paste the Access Token into local.properties.
```

Comment:
3. Add the access token string copied from the console to a variable named `TWILIO_ACCESS_TOKEN` in your **local.properties** file.

```
TWILIO_ACCESS_TOKEN=abcdef0123456789
```